apps/package.json

```diff
@@ -26,12 +26,15 @@
     "build:stats": "node ./script/build-stats.js",
     "prestorybook": "curl -o build/package/css/application.css http://localhost:3000/assets/application.css",
     "storybook": "start-storybook -p 9001 -s build/package/",
-    "storybook:deploy": "./script/deploy-storybook.sh"
+    "storybook:deploy": "./script/deploy-storybook.sh",
+    "codemod:es6-modules": "./script/codemod-es6-modules.sh",
+    "codemod:move-file": "./script/codemod-move-file.sh"
```

Comment:
For one off scripts (especially if they require a different node version) I wonder if it would make more sense not to expose them here.